desktop/src/app/components/navbar/warnings/modals/fix-outliers-modal.component.ts

```diff
@@ -53,8 +56,40 @@ export class FixOutliersModalComponent {
 
         this.projectDocument = await this.datastore.get('project');
         this.valuelist = await this.getValuelist(this.document, this.field);
+        this.foundDocuments = await this.datastore.find({
```

Comment:
Fetching the documents can be done in "prepareReplaceAll" as "foundDocuments" is only used in that function